### DISCUSSION TITLE
Running into issues when nesting routers in the `appRouter` to create separate namespaces with ExpressJS

### DISCUSSION THREAD

**Discussion by 0xtygra** [trpc/trpc#3361] [Q&A]
Creating this as a discussion as the issue is likely on my end!

Here's what I have for my `appRouter`
![image](https://user-images.githubusercontent.com/24663638/206624256-24a14d95-8d40-4ae6-b4a4-6b8a46d93c04.png)
my server's `main.ts`
![image](https://user-images.githubusercontent.com/24663638/206624303-ca337bf3-fb47-42e7-8ce5-ef816313c14a.png)
and my `trpc.ts`
![image](https://user-images.githubusercontent.com/24663638/206624343-9b0764bf-6d96-470a-a60a-bd3ebe0778b5.png)

When running this server, `localhost:8000/trpc/get` correctly returns me my `"hi"` message whereas `localhost:8000/trpc/test/get` gives me a 404. Am I not nesting the routers correctly? The docs + tRPC x Express example app show that this should create a /test namespace in this case. Really appreciate any help!

![image](https://user-images.githubusercontent.com/24663638/206624465-c328d247-064f-4dbc-b65d-c35213ac28dd.png)
![image](https://user-images.githubusercontent.com/24663638/206624609-9a19b6de-18d2-4b0e-9ed6-14540ea1504c.png)

**Comment by veikovx** [comment]
Instead of `localhost:8000/trpc/test/get ` you need to do `localhost:8000/trpc/test.get` as tRPC nested routes are accessed via `.` 

When you do `app.user('trpc', trpcExpress......` you effectively make tRPC the handler for that path, and everything under it, in Express. So everything following that path does not need to follow Express logic

**Reply by DivyanshuPandey1702** [reply]
but i want it by / what can i do then, is there a way to change . to /

**Reply by ig4e** [reply]
> but i want it by / what can i do then, is there a way to change . to /

Currently no
